jaxrs-clients/src/test/java/com/palantir/remoting2/jaxrs/JaxRsClientGuavaOptionalHandlingTest.java

```diff
@@ -90,7 +91,7 @@ public void testRegularPathParam() throws Exception {
 
     @Test
     public void testAbsentQuery() throws Exception {
-        proxy.query(Optional.<String>absent(), "str2");
+        proxy.query(Optional.absent(), "str2");
```

Comment:
because this explicitly tests the guava functionality. there's a corresponding thing for java8